### DISCUSSION TITLE
Ikev1 Disable in strongswan

### DISCUSSION THREAD

**Discussion by dewilllll** [strongswan/strongswan#1235] [Q&A]
Hi,
We want to disable ikev1 for all the connections. So is there any way to disable it on the storngswan level? We already know how to do it in swanctl.conf, but we want to make it generic to all connections /conf.

**Comment by tobiasbrunner** [comment]
What's the problem with doing this via swanctl.conf?

If you are building strongSwan yourself, you can compile it without IKEv1 support (`--disable-ikev1`).
